src/app/core/services/githubevent.service.ts

```diff
@@ -55,19 +55,19 @@ export class GithubEventService {
     );
   }
 
-  setLastModifiedTime(lastModified: string) {
+  private setLastModifiedTime(lastModified: string): void {
```

Comment:
In addition to the unit tests, I added the private modifier for these methods since they do not need to be accessed elsewhere.